utils.py

```diff
@@ -123,7 +120,7 @@ def createWorker(
         docker_host=private_config["private"]["docker_workers"][worker_name],
         image=image_str,
         dockerfile=dockerfile_str,
-        tls=tls,
+        tls=None,
```

Comment:
Why?

Reply:
It was always passed as none. See the deletion from above. I just made it obvious.

Reply:
https://github.com/MariaDB/buildbot/pull/637/files/d78def1836c0638a1e7a56ccc4c316e60d7eab18#diff-09d0cbaf891d623ac245ccf84485a687b554217349ca55557c60b33234b969aeL89

Reply:
Ok for most of the worker, this is fine since we use wireguard, no reason to use extra layer of security. But we have still a bunch of workers that communicate on the public IP, @RazvanLiviuVarzaru now that you have a better overview could you please double check that point?

Reply:
Indeed it was always None as default on every function call.
Shouldn't affect existing workers configured via `createWorker` function.

I propose to set tls with default value None as a function argument, for future scenarios where wireguard is not an option.
Having it as an argument is clear enough for the caller also.